test/stringify.js

```diff
@@ -18,6 +18,11 @@ test('stringify()', function (t) {
         st.end();
     });
 
+    t.test('adds query prefix', function (st) {
+        st.equal(qs.stringify({ a: 'b' }, { addQueryPrefix: true }), '?a=b');
+        st.end();
+    });
```

Comment:
Hi, regarding the new `addQueryPrefix` option, I noticed the following situation:
```
> qs.stringify({ a: 1, b: 2}, { addQueryPrefix: true })
'?a=1&b=2'
> qs.stringify({}, { addQueryPrefix: true})
'?'
```
In my use case I don't want to keep the lonely `?` if I'm stringifying an empty object.

What do you think about it?